scipy/optimize/tests/test_differentiate.py

```diff
@@ -64,7 +64,7 @@ def f(x, *args, **kwargs):
         assert_equal(res.df.shape, shape)
 
         ref_error = [ref.error for ref in refs]
-        assert_allclose(res.error.ravel(), ref_error, atol=5e-15)
+        assert_allclose(res.error.ravel(), ref_error, atol=1e-12)
```

Comment:
I think we can be less aggressive here
```suggestion
        assert_allclose(res.error.ravel(), ref_error, atol=1e-14)
```